python/pyspark/pandas/plot/core.py

```diff
@@ -22,8 +22,6 @@
 from pandas.core.base import PandasObject
 from pandas.core.dtypes.inference import is_integer
 
-from pyspark.ml.feature import Bucketizer
```

Comment:
those plotting functions are actually not support in spark connect, since the underlying functions are built atop classic mllib.
we'd reimpl them to support connect. cc @xinrong-meng